cli/lib/kontena/cli/services/deploy_command.rb

```diff
@@ -7,14 +7,19 @@ class DeployCommand < Kontena::Command
     include ServicesHelper
 
     parameter "NAME", "Service name"
-    option '--force-deploy', :flag, 'Force deploy even if service does not have any changes'
+    option '--force', :flag, 'Force deploy even if service does not have any changes'
+    option '--force-deploy', :flag, '[DEPRECATED: use --force]'
 
     def execute
       require_api_url
       token = require_token
       service_id = name
       data = {}
-      data[:force] = true if force_deploy?
+      data[:force] = true if force? || force_deploy? # deprecated
+      if force_deploy?
+        print "[WARNING]".colorize(:yellow)
```

Comment:
`Kontena::CLI::Common` module (included already) includes `warning` method you can use here: https://github.com/mattdavenport/kontena/blob/master/cli/lib/kontena/cli/common.rb#L93
It will output the warning message to STDERR with the correct formatting and color, so you don't need to worry about that.